proto/decentralized-vault/decentralized-vault.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+package ununifi.decentralizedvault;
+
+import "gogoproto/gogo.proto";
+import "decentralized-vault/params.proto";
+
+option go_package = "github.com/UnUniFi/chain/x/decentralized-vault/types";
+
+message TransferRequest {
+  string owner       = 1;
+  string eth_address = 2;
+  string nft_id      = 3;
+}
+
+message TransferRequestNft {
+  string id       = 1;
+  string uri      = 2;
+  string uri_hash = 3;
+  string owner    = 4 [
```

Comment:
Is this transfer request to this owner? (Probably rename will be helpful for clarity)